CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.0]
+### Added
+- Add new input to support customising the Slack channel ([#40](https://github.com/MetaMask/action-npm-publish/pull/40))
+### Fixed
+- Fix functionality for subteams ([#44](https://github.com/MetaMask/action-npm-publish/pull/44))
```

Comment:
Missing newline, and we should call attention to the breaking part of this change (and maybe list it separately).

e.g.
```suggestion

### Removed
- **BREAKING:** Remove `target-name` option ([#44](https://github.com/MetaMask/action-npm-publish/pull/44))
  - `target-name` used to support tagging individuals in Slack announcements. This option has been replaced by a `subteam` option just for teams; tagging individuals is no longer supported

### Fixed
- Fix functionality for subteams ([#44](https://github.com/MetaMask/action-npm-publish/pull/44))
```